sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/ConditionalStructureSameCondition.CSharp9.cs

```diff
@@ -0,0 +1,51 @@
+﻿object o;
+int i;
+
+void SimpleTest()
+{
+    if (o is not null) // Secondary [line11]
```

Comment:
I think it would be better to use something else for the labels than line number since it can get hard to maintain.

If we want to add one more case for which an "using" is needed we would need to update all the labels.